.vscode/launch.json

```diff
@@ -0,0 +1,58 @@
+// https://code.visualstudio.com/docs/editor/debugging#_launchjson-attributes
+// https://code.visualstudio.com/docs/editor/variables-reference
+// https://github.com/microsoft/vscode-recipes/tree/master/debugging-Ruby-on-Rails
+// https://github.com/rubyide/vscode-ruby/blob/main/docs/debugger.md
+// Debug configurations requires..
+// the solargraph, debase, and ruby-debug-ide gems to be (bundle) installed, and
+// the castwide.solargraph, and rebornix.ruby extensions to be installed.
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug Rails server",
+      "type": "Ruby",
+      "request": "launch",
+      "cwd": "${workspaceRoot}",
+      "useBundler": true,
+      "pathToBundler": "${env:BUNDLER_PATH}",
+      "pathToRDebugIDE": "${env:RUBY_DEBUG_IDE_PATH}",
+      "program": "${workspaceRoot}/bin/rails",
+      "args": [
+        "server",
+        "-b",
+        "0.0.0.0",
+        "-p",
+        "3000"
+      ],
+      "env": {
+        "RAILS_ENV": "development"
+      }
+    },
+    {
+      "name": "Debug RSpec - selected line, active file",
+      "type": "Ruby",
+      "request": "launch",
+      "cwd": "${workspaceRoot}",
+      "useBundler": true,
+      "pathToBundler": "${env:BUNDLER_PATH}",
+      "pathToRDebugIDE": "${env:RUBY_DEBUG_IDE_PATH}",
+      "program": "/home/chemotion-dev/.asdf/installs/ruby/2.7.6/bin/rspec", // "program" doesn't support env variable substitution
```

Comment:
Shouldn't a ```bundle exec rspec``` suffice? you are already in workspace root and as long as the devcontainer does not use some strange PATH shenanigans, the regular way of invoking rspec should be fine

Reply:
Do you mean changing the value for "program" from
`"/home/chemotion-dev/.asdf/installs/ruby/2.7.6/bin/rspec"`
to
`"bundle exec rspec"`?

Reply:
What about the useBundler option in line 36?

Reply:
The `useBundler` option is documented here: https://github.com/rubyide/vscode-ruby/wiki/2.-Launching-from-VS-Code.

The `launch.json` debugging configurations are somewhat intransparent. What happens is that VSCode infers from `type: "Ruby"` that it's supposed to debug a "program" using the `ruby-debug-ide`. In case the latter gem has been bundle-installed, the configuration requires setting the `useBundler` option to `true`.